Binoculars.csproj

```diff
@@ -148,21 +148,23 @@
       <SourcePkg Include="$(ProjectDir)credentials.json" />
       <SourceExtension Include="$(TargetDir)*ExtensionDefinition.xml" />
     </ItemGroup>
+    <!--Delete any previous build folder -->
+    <RemoveDir Directories="$(TargetDir)$(ProjectName)" />
     <!--Copying to Build Folder-->
     <Copy SourceFiles="@(SourceDlls)" DestinationFolder="$(TargetDir)$(ProjectName)\bin\" />
     <Copy SourceFiles="@(SourcePdbs)" DestinationFolder="$(TargetDir)$(ProjectName)\bin\" />
     <Copy SourceFiles="@(SourcePkg)" DestinationFolder="$(TargetDir)$(ProjectName)" />
-    <Copy SourceFiles="@(SourceExtension)" DestinationFolder="$(TargetDir)\$(ProjectName)\extra" />
+    <Copy SourceFiles="@(SourceExtension)" DestinationFolder="$(TargetDir)$(ProjectName)\extra" />
     <Message Importance="High" Text="++++++++++++++++++++++++++++++++++++++" />
-    <Message Importance="High" Text="Built to $(TargetDir)\$(ProjectName)" />
+    <Message Importance="High" Text="Built to $(TargetDir)$(ProjectName)" />
     <ItemGroup>
-      <SourcePackage Include="$(TargetDir)\$(ProjectName)\**\*" />
+      <SourcePackage Include="$(TargetDir)$(ProjectName)\**\*" />
     </ItemGroup>
     <PropertyGroup>
       <!--Copy to Dynamo sandbox for testing -->
-      <DeployFolder Condition="'$(Configuration)' == 'Debug'">$(AppData)\Dynamo\Dynamo Core\2.0\packages\$(ProjectName)</DeployFolder>
+      <DeployFolder Condition="'$(Configuration)' == 'Debug' Or '$(Configuration)' == 'Release'">$(AppData)\Dynamo\Dynamo Core\2.0\packages\$(ProjectName)</DeployFolder>
```

Comment:
This is the import change.

The in code comments probably also need revising as it's incorrect that Sandbox is for testing and Revit is for publishing?